js/index.js

```diff
@@ -1,227 +0,0 @@
-    { x_pos: 200 + width * 3, y_pos: 80, size: 80 },
-    { x_pos: 500 + width * 3, y_pos: 120, size: 80 },
-    { x_pos: 750 + width * 3, y_pos: 150, size: 80 },
-  ];
-
-  loop();
-}
-
-function draw() {
-  if (gameStarted) {
-    background(color(100, 155, 255));
-    noStroke();
-    textSize(20);
-    fill("white");
-    text(`Lives: ${lives}`, 20, 20);
-
-    // Draw clouds
-    clouds.forEach((cloud) => {
-      drawCloud(cloud);
-      // Movement for clouds, whilst on the visible part of the stage
-      if (cloud.x_pos < 750 + width * 3) {
-        cloud.x_pos += 0.3;
-        // When clouds go off the visible part of the stage, their x_pos is set to the start of the stage
-      } else {
-        cloud.x_pos = 100 - width;
-      }
-    });
-
-    platforms.forEach((platform) => platform.draw());
-
-    if (enemies) {
-      // Draw ball n chains
-      for (let i = 0; i < enemies.length; i++) {
-        enemies[i].draw();
-        enemies[i].updateGC_Pos(player.x, player.y);
-        const isContact = enemies[i].checkContact(player.x, player.y);
-        if (isContact) {
-          if (lives > 1) {
-            lives--;
-            startGame();
-          } else if (lives === 1) {
-            lives--;
-            startGame();
-          }
-        }
-      }
-    }
-
-    player.draw();
-
-    //ScrollWindow
-    if (keys.right.pressed && player.x < 400) {
-      player.velX = 5;
-    } else if (keys.left.pressed && player.x > 100) {
-      player.velX = -5;
-    } else {
-      player.velX = 0;
-      if (keys.right.pressed) {
-        platforms.forEach((platform) => (platform.x -= 3));
-        enemies.forEach((enemy) => (enemy.vector.x -= 3));
-      } else if (keys.left.pressed) {
-        platforms.forEach((platform) => (platform.x += 3));
-        enemies.forEach((enemy) => (enemy.vector.x += 3));
-      }
-    }
-    //Loose
-    if ((!player.isOnplatform() && player.y > height) || lives <= 0) {
-      if (lives > 0) {
-        lives -= 1;
-        startGame();
-      } else {
-        drawMessage("Game Over!", "lost");
-      }
-    }
-    //Win
-    if (player.isOnCheckPoint()) {
-      drawMessage("Win!", "win");
-    }
-  }
-}
-
-function drawMessage(message, gStatus) {
-  textSize(40);
-  text(message, width / 2 - 100, height / 2);
-  textSize(20);
-  text("Press Space to restart", width / 2 - 150, height / 2 + 20);
-  gameStatus = gStatus;
-  noLoop();
-}
-
-function keyPressed() {
-  if (keyCode === UP_ARROW && player.isOnplatform()) {
-    player.velY -= 20;
-  }
-  if (keyCode === LEFT_ARROW) {
-    keys.left.pressed = true;
-  }
-  if (keyCode === RIGHT_ARROW) {
-    keys.right.pressed = true;
-  }
-  if (
-    (keyCode === 32 && gameStatus === "win") ||
-    (keyCode === 32 && gameStatus === "lost")
-  ) {
-    lives = 2;
-    startGame();
-    loop();
-  }
-}
-
-function keyReleased() {
-  switch (keyCode) {
-    // case 38: //up
-    //   break;
-    // case 40: //down
-    //   break;
-    case 37: //left
```

Comment:
same here. instead of comments, just use variables. you could even group them within a "keyCodes" object. and then say. keyCode.space for example.